etcdserver/server.go

```diff
@@ -1689,7 +1689,7 @@ func (s *EtcdServer) getLead() uint64 {
 	return atomic.LoadUint64(&s.lead)
 }
 
-func (s *EtcdServer) leaderChangedNotify() <-chan struct{} {
+func (s *EtcdServer) LeaderChangedNotify() <-chan struct{} {
```

Comment:
Please add it to the 'Server' interface. 

Please comment on the semantic:
  1. that this channel is being closed when the leadership changes.
  2. so the new channel needs to be obtained for each term. 
  3. That you can loose some consecutive channel changes using this API.

Reply:
done